docs/setup/pick-right-solution.md

```diff
@@ -164,14 +164,14 @@ GCE                  | CoreOS       | CoreOS | flannel     | [docs](/docs/gettin
 Vagrant              | CoreOS       | CoreOS | flannel     | [docs](/docs/getting-started-guides/coreos/)                                 |  Community ([@pires](https://github.com/pires), [@AntonioMeireles](https://github.com/AntonioMeireles))
 Bare-metal (Offline) | CoreOS       | CoreOS | flannel     | [docs](/docs/getting-started-guides/coreos/bare_metal_offline/)              |  Community ([@jeffbean](https://github.com/jeffbean))
 CloudStack           | Ansible      | CoreOS | flannel     | [docs](/docs/getting-started-guides/cloudstack/)                             |  Community ([@sebgoa](https://github.com/sebgoa))
-Vmware vSphere       | Saltstack    | Debian | OVS         | [docs](/docs/getting-started-guides/vsphere/)                                |  Community ([@imkin](https://github.com/imkin))
+VMware vSphere       | Saltstack    | Debian | OVS         | [docs](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/)  |  Community ([@imkin](https://github.com/imkin))
```

Comment:
Let's update this to latest for other fields too

Reply:
Addressed in 6733909.